src/reducers/tests/ui.js

```diff
@@ -28,4 +28,16 @@ describe("ui reducer", () => {
     expect(updatedState.frameworkGroupingOn).toBe(value);
     expect(prefs.frameworkGroupingOn).toBe(value);
   });
+
+  it("gets the search results", () => {
+    const state = State();
+    expect(getSearchResults({ ui: state })).toBe(state.searchResults);
+  });
+
+  it("gets the symbol search results", () => {
+    const state = State();
+    expect(getSymbolSearchResults({ ui: state })).toBe(
+      state.symbolSearchResults
+    );
```

Comment:
I guess false / true should be replaced with something else? or maybe we drop these tests for now?